vim_template/langs/python/python.vim

```diff
@@ -17,6 +17,9 @@ let g:jedi#show_call_signatures = "0"
 let g:jedi#completions_command = "<C-Space>"
 let g:jedi#smart_auto_mappings = 0
 
+" black
+autocmd BufWritePre *.py execute ':Black'
```

Comment:
`Ale` is already part of the project and can be configures to use `black` as fixer. No new command is necessary and this might be the way to go.

https://github.com/dense-analysis/ale/blob/c4bdf165caf99706f439ecbe1a1d1865d53545a8/autoload/ale/fixers/black.vim

https://github.com/dense-analysis/ale/blob/8b46fa3ee716485b4dd7c3f86c9302a140be6bfa/doc/ale-python.txt#L122

Reply:
yep, read more in https://black.readthedocs.io/en/stable/editor_integration.html#vim